test/test_distributions.py

```diff
@@ -225,8 +226,14 @@ def ref_log_prob(idx, val, log_prob):
             self.assertEqual(log_prob, math.log(prob if val else 1 - prob))
 
         self._check_log_prob(Bernoulli(p), ref_log_prob)
+        self._check_log_prob(Bernoulli(logits=p.log() - (-p).log1p()), ref_log_prob)
```

Comment:
why not just `p.log()`?

Reply:
There are two common definitions of logits, one corresponding to log-likelihood and one corresponding to log-odds. The `logits` arg to `Categorical` is the former. The `logits` arg to `Bernoulli` is the latter and is equal to `log(p / (1-p))`. If we were to implement `Bernoulli` via `Categorical`, the logits would be a pair of numbers `[log(p), log(1-p)]` but `Bernoulli` can simply use the log-ratio of the two.

Reply:
That makes sense, but I think it would be good to use a different name (perhaps `log_odds`), so that it's immediately clear that it's expected to be a different thing. I can merge it as is if you think that's unnecessary, but it feels like a safer interface to me.

Reply:
I was trying to be consistent with how Tensorflow defines it, but we could change it to log odds for Bernoulli too.

Reply:
I don't think keeping it 100% consistent is worth it, especially that it's not a silent difference. It will be immediately clear that you have to use a different name, or you'll get an error.

I'm a bit worried of calling two semantically different things using the same name (if you don't at least have a type system), because that's a recipe for an easy failure that will take a long time to debug.

Reply:
Actually, `logits` and `log odds` refer to the same thing, namely, `log (p/(1-p))`. The issue is that for the multidimensional case, `logits` is used to loosely refer to log of probability (possibly unnormalized). So rather than changing `logits` to `log_odds` in Bernoulli, we could change the `logits` in Categorical to `log_probs` instead (that sounds similar to the `log_prob` method though) to resolve this potential confusion.

@fritzo - do you have any thoughts / preferences on the naming?

Reply:
Tensorflow uses `logits` informally throughout to refer to both log-odds and log-probs. In an effort to maximize compatibility with [tf.distributions.Categorical](https://www.tensorflow.org/api_docs/python/tf/distributions/Categorical), I recommend we name this parameter `logits`.

Reply:
Alright, let's leave it as is